docs/data-sources/grafana_dashboard.md

```diff
@@ -0,0 +1,48 @@
+---
+
+# generated by https://github.com/hashicorp/terraform-plugin-docs
+
+page_title: "coralogix_grafana_dashboard Data Source - terraform-provider-coralogix"
+subcategory: ""
+description: "Hosted grafana dashboard.
+
+* Official documentation https://grafana.com/docs/grafana/latest/dashboards/
+* HTTP API https://grafana.com/docs/grafana/latest/http_api/dashboard/"
+
+---
+
+# coralogix_grafana_dashboard (Data Source)
+
+## Example Usage
+
+```hcl
+    data "coralogix_grafana_dashboard" dashboard {
```

Comment:
This is different then what we discussed.
We wanted `hosted_dashboard` and that should be used for both grafana and opensearch